git/libgit2/go.mod

```diff
@@ -31,6 +31,8 @@ replace github.com/go-git/go-billy/v5 => github.com/go-git/go-billy/v5 v5.3.2-0.
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
 	github.com/ProtonMail/go-crypto v0.0.0-20221026131551-cf6655e29de4
+	github.com/armon/go-socks5 v0.0.0-20160902184237-e75332964ef5
+	github.com/elazarl/goproxy v0.0.0-20220529153421-8ea89ba92021
```

Comment:
`github.com/elazarl/goproxy` seems unmaintained, but mostly self-contained. It would be good to find an alternative at some point.